proxygen/deps.sh

```diff
@@ -22,6 +22,9 @@ set -e
 start_dir=$(pwd)
 trap 'cd $start_dir' EXIT
 
+folly_rev=`cat $start_dir/../build/deps/github_hashes/facebook/folly-rev.txt|sed 's/Subproject commit //'`
```

Comment:
Our linter says:

(style) Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
(info) Double quote to prevent globbing and word splitting.

Also: (style) Use $(..) instead of legacy ...
but this is all over the file so we can clean that up later.

Reply:
which linter are you using?

Reply:
It's probably something we wrote